src/toRedactor.tsx

```diff
@@ -362,7 +403,18 @@ export const toRedactor = (jsonValue: any,options?:IJsonToHtmlOptions) : string
           delete attrsJson['content-type-uid']
           attrsJson['sys-style-type'] = allattrs['display-type']
           delete attrsJson['display-type']
-        } else if (attrsJson['type'] === "asset") {
+        } 
+        else if (attrsJson["display-type"]) {
```

Comment:
I see one concern here...what if attrsJson object has both `display-type` and `type: asset`, then the next else here below this won't ever be reachable, just make sure that case is handled, or else we can make the if condition here more specific.

Reply:
I have now added new conditions inside the existing asset check to make sure all the previous properties are still getting applied to the img tag